themes/default/content/templates/static-website/_index.md

```diff
@@ -6,4 +6,28 @@ meta_image: meta.png
 weight: 1
 ---
 
-Pulumi program templates that make it easy to deploy static websites on AWS, Azure, or Google Cloud Platform.
+Pulumi program templates are the fastest way to deploy static websites on AWS, Azure, or Google Cloud Platform. Templates come with predefined infrastructure as code so you can get started instantly.
```

Comment:
link '[infrastructure as code](https://www.pulumi.com/what-is/what-is-infrastructure-as-code/)'